### PR TITLE
Add message sender to AI that happens every time AI makes a move

### PR DESCRIPTION
**Pull request by KIsForCookie**

### CODE REVIEW

File: app/src/main/java/com/bestCatHustlers/sukodublitz/GameAI.java [hunk 1]
```diff
@@ -18,12 +22,16 @@
     private int delayMs;
     private BoardGame game;
     private String id;
-    GameAI(BoardGame game, int delayMs, String id)
+    private GamePresenter presenter;
+    public final int UPDATE_BOARD = 1234;
```
Comment (yufeiyang-dev) [inline app/src/main/java/com/bestCatHustlers/sukodublitz/GameAI.java]: what does this `int` value stand for? (UPDATE_BOARD doesn't give me too much clue)

**Review by yufeiyang-dev — DISMISSED** [review]

File: app/src/main/java/com/bestCatHustlers/sukodublitz/GameAI.java (hunk 1 above)
Comment (xjchong) [reply app/src/main/java/com/bestCatHustlers/sukodublitz/GameAI.java]: Going to have to agree with @Stevenfly on this, @KIsForCookie. Some of the naming could be clearer haha 😅

File: app/src/main/java/com/bestCatHustlers/sukodublitz/GameAI.java [hunk 2]
```diff
@@ -61,4 +70,10 @@ public void run()
             }
         }
     }
+
+    private void sendMessage()
```
Comment (xjchong) [inline app/src/main/java/com/bestCatHustlers/sukodublitz/GameAI.java]: This should probably be renamed to something more descriptive too.

**Review by xjchong — DISMISSED** [review]
I know the backend code is mostly only written to by you, but the naming conventions could use some improvement. 😅

File: app/src/main/java/com/bestCatHustlers/sukodublitz/GameAI.java (hunk 1 above)
Comment (KIsForCookie) [reply app/src/main/java/com/bestCatHustlers/sukodublitz/GameAI.java]: You need to give some kind of integer message contents with the message.

**Review by yufeiyang-dev — APPROVED** [review]

**Review by xjchong — APPROVED** [review]

**Pull request merged**